manifests/permissive.pp

```diff
@@ -10,7 +10,8 @@
 #   }
 #
 define selinux::permissive (
-  $context,
+  String $context,
```

Comment:
I think the param should default to `$title` if we're at it changing.

what about naming it `seltype` as in the other renewed types? I think this would be consistent.